Makefile

```diff
@@ -83,6 +84,9 @@ destroy_all: \
 	destroy_gocd \
 	destroy_elk \
 	destroy_worker \
+	destroy_application \
+	destroy_snapshot \
+	destroy_amicreation \
```

Comment:
@hazim1093 how about renaming the module to "amicreator" or "animator" or "baker" etc.